src/connector/src/sink/clickhouse.rs

```diff
@@ -562,21 +601,35 @@ impl ClickHouseSinkWriter {
             }
             match op {
                 Op::Insert | Op::UpdateInsert => {
-                    if self.clickhouse_engine.get_sign_name().is_some() {
+                    if self.clickhouse_engine.is_collapsing_engine() {
```

Comment:
I modified this @xxhZs as this was calling a `to_string()` which in Rust I believe incurs a memory copy...and this would be on a per-row basis. We aren't using the string in either the sign or delete, so we should instead just check its persence.